src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -258,6 +267,27 @@ export default {
 			return this.isInCall && this.pollsStore.isNewPoll(this.message.messageParameters.object.id)
 		},
 
+		isCallEndedMessage() {
+			return this.message.systemMessage === 'call_ended' || this.message.systemMessage === 'call_ended_everyone'
+		},
+
+		conversation() {
+			return this.$store.getters.conversation(this.message.token)
+		},
+
+		showEventConversationActions() {
+			return !this.isInCall && !this.isSidebar && this.$store.getters.isModeratorOrUser
+				&& this.conversation.objectType === CONVERSATION.OBJECT_TYPE.EVENT
+				&& getEventTimeRange(this.conversation).end < Date.now()
```

Comment:
maybe use `start < now + 5 or 10 minutes` here? If meeting ends 5 minutes before the endtime, we won't see these buttons.

Also I'd move it to the bottom of the list to be the last check